tests/tool_cloudmetrics_collect_metrics_test.php

```diff
@@ -126,13 +126,32 @@ public function test_execute(string $timestr, array $meta, array $expected) {
         $mock->expects($this->once())
             ->method('receive')
             ->with($expected);
+
+        $this->disable_custom_metrics();
         $task = new helper_collect_metrics_task($mock);
         $task->set_time($time);
         ob_start();
         $task->execute();
         ob_end_clean();
     }
 
+    /**
+     * This method gets all metrics available and disable those out of the builtin metrics.
+     *
+     * @return void
```

Comment:
Can remove this return annotation :)

Reply:
Thanks for the feedback. The metrics are required to measured in sync with each other reason why the test will fail again if I disable all of them. For example, the last set of data provider sends the `new_user_metric`  with FREQ_5MIN to be measured in the period of time 1 day, and those metrics with default FREQ_5MIN and up to FREQ_DAY will also be measured at this time (which is the case for all builtin metrics). And so on. 

By default, our custom metrics are FREQ_5MIN meaning they should be process by the task as well. So, I maybe need to find another approach around.

Reply:
We could try something like the following:

```
foreach (metric\manager::get_metrics(true) as $metric) {
    if (!in_array($metric->get_name(), $expected)) {
        $metric->set_enabled(false);
    }
}
```

So now, all metrics except the ones that the testcase cares about are disabled. We could also remove the call to `$metric->set_enabled(true)` in the `foreach ($meta as $metric => $data)` loop.

Reply:
Thanks for the suggestion @cameron1729 I have added the fixes and also updated some other code sections to comply with phpcs rules. 
Please review it when available.